### PR TITLE
Python 3: re-update for autotest commands removal

### PR DESCRIPTION
**Pull request by hahakiki2010**
Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def check_dump_format(dump_image_format, dump_file):
             return True
         else:
             file_cmd = "file %s" % dump_file
-            ret = process.run(file_cmd, shell=True)
+            ret = process.run(file_cmd, allow_output_check='combined', shell=True)
             status, output = ret.exit_status, ret.stdout.strip()
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: need change to stdout_text

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def check_bypass(dump_file, result_dict):
                 continue
             cmd2 = ("cat /proc/$(%s |awk '/libvirt_i/{print $2}')/fdinfo/1"
                     "|grep flags|awk '{print $NF}'" % cmd1)
-            ret = process.run(cmd2, shell=True)
+            ret = process.run(cmd2, allow_output_check='combined', shell=True)
             status, output = ret.exit_status, ret.stdout.strip()
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: need change to stdout_text

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def run(test, params, env):
 
     cmd = "qemu-img create -f %s %s %s" % (image_format, image_path,
                                            initial_disk_size)
-    ret = process.run(cmd, shell=True)
+    ret = process.run(cmd, allow_output_check='combined', shell=True)
     status, output = (ret.exit_status, ret.stdout.strip())
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py]: need change to stdout_text

File: libvirt/tests/src/multifunction.py [hunk 4]
```diff
@@ -170,7 +170,7 @@ def attach_additional_device(vm_name, disksize, targetdev, params):
     logging.info("Attaching disk...")
     disk_path = os.path.join(data_dir.get_tmp_dir(), targetdev)
     cmd = "qemu-img create %s %s" % (disk_path, disksize)
-    ret = process.run(cmd, shell=True, allow_output_check='combine')
+    ret = process.run(cmd, shell=True, allow_output_check='combined')
     status, output = ret.exit_status, ret.stdout.strip()
```
Comment (chunfuwen) [inline libvirt/tests/src/multifunction.py]: need change to stdout_text

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py (hunk 1 above)
Comment (hahakiki2010) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: There will a dedicated PR later to replace all the stdout to stdout_text. For this PR, could we just focus on combination of stderr and stdout to the same output file ?

File: libvirt/tests/src/virsh_cmd/domain/virsh_dump.py (hunk 2 above)
Comment (hahakiki2010) [reply libvirt/tests/src/virsh_cmd/domain/virsh_dump.py]: same as above

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py (hunk 3 above)
Comment (hahakiki2010) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py]: same as above

File: libvirt/tests/src/multifunction.py (hunk 4 above)
Comment (hahakiki2010) [reply libvirt/tests/src/multifunction.py]: same as above

**Pull request merged**